mobile/lib/modules/activities/providers/activity.provider.dart

```diff
@@ -95,7 +95,11 @@ class ActivityStatisticsNotifier extends StateNotifier<int> {
   }
 
   Future<void> fetchStatistics() async {
-    state = await _activityService.getStatistics(albumId, assetId: assetId);
+    final count =
+        await _activityService.getStatistics(albumId, assetId: assetId);
+    if (mounted) {
```

Comment:
Where does this `mounted` variable come from?

Reply:
It is part of `StateNotifier`